### ISSUE TITLE
Parsing error

### ISSUE THREAD

**Issue by under-score** [bgcarlisle/cthist#23]
```
Error downloading NCT ID: NCT02020174                                                                                       
Here's the original error message:
Error in `dplyr::rename()`:
! Can't rename columns that don't exist.
✖ Column `version` doesn't exist.
```
thank you for looking into this.

**Comment by bgcarlisle** [comment]
It looks like this is happening because there's no history listed at all on ClinicalTrials.gov for the NCT number you mention:

https://clinicaltrials.gov/study/NCT02020174?tab=history

Here's an example of another entry that *does* have a history:

https://clinicaltrials.gov/study/NCT06194526?tab=history

Honestly, it's confusing to me how there's nothing at all for that NCT number, and I think it must be a bug on ClinicalTrials.gov's end

My recommendation for the time being is to take that one NCT number out of the set of trials you're trying to get data for, and the rest should work (if it's the only one you're interested in, you might need to contact the people at ClinicalTrials.gov to figure out why the history is entirely missing)

**Comment by under-score** [comment]
I agree, seems like a ClinicalTrials.gov bug. Unfortunately NCBI desk was not very helpful in the past, so I am just filtering for this error ahead of the clinicaltrials_gov_download() function.

**Comment by bgcarlisle** [comment]
Checking in on this, it appears that [NCT02020174](https://clinicaltrials.gov/study/NCT02020174?tab=history) has found its history item after all

When I run the following:

```
> hv <- clinicaltrials_gov_download("NCT02020174")
```

I get a well-formed data frame in `hv`

If you notice any other errors, please do let me know
